src/app/buy-bitclout-page/buy-bitclout/buy-bitclout.component.ts

```diff
@@ -497,8 +488,16 @@ export class BuyBitcloutComponent implements OnInit {
 
   ngOnInit() {
     window.scroll(0, 0);
-    this.showAreYouReady =
-      this.appData != null && this.appData.loggedInUser != null && this.appData.loggedInUser.BalanceNanos === 0;
+
+    // Add extra tabs
+    if (this.globalVars.showBuyWithUSD) {
+      this.buyTabs.push(BuyBitcloutComponent.BUY_WITH_USD);
+      this.activeTab = BuyBitcloutComponent.BUY_WITH_USD;
```

Comment:
The default state when you have btc, wyre, and eth enabled looks weird now since BUY_WITH_USD is in the middle and selected.  Maybe make BUY_WITH_USD the first tab when it's enabled?
![image](https://user-images.githubusercontent.com/81658108/133547710-c6ac8064-f638-4b4f-80fe-684cbb4f8cda.png)